plugins/GigPlayer/GigPlayer.cpp

```diff
@@ -408,17 +407,17 @@ void GigInstrument::play( sampleFrame * _working_buffer )
 	}
 
 	// Fill buffer with portions of the note samples
-	for( QList<GigNote>::iterator it = m_notes.begin(); it != m_notes.end(); ++it )
+	for(auto & m_note : m_notes)
 	{
 		// Only process the notes if we're in a playing state
-		if( !( it->state == PlayingKeyDown ||
-				it->state == PlayingKeyUp ) )
+		if( !( m_note.state == PlayingKeyDown ||
+				m_note.state == PlayingKeyUp ) )
 		{
 			continue;
 		}
 
-		for( QList<GigSample>::iterator sample = it->samples.begin();
-				sample != it->samples.end(); ++sample )
+		for( QList<GigSample>::iterator sample = m_note.samples.begin();
+				sample != m_note.samples.end(); ++sample )
```

Comment:
Interestingly, it hasn't converted this loop. Perhaps this is because it was already changed by converting the outer loop, in which case multiple passes may be required.